Sources/Turf/Geometry.swift

```diff
@@ -4,21 +4,129 @@ import CoreLocation
 #endif
 
 
-public enum GeometryType: String {
+public enum GeometryType: String, Codable, CaseIterable {
     case Point
     case LineString
     case Polygon
     case MultiPoint
     case MultiLineString
     case MultiPolygon
+    case GeometryCollection
     
-    static let allValues: [GeometryType] = [.Point, .LineString, .Polygon, .MultiPoint, .MultiLineString, .MultiPolygon]
+//    static let allValues: [GeometryType] = [.Point, .LineString, .Polygon, .MultiPoint, .MultiLineString, .MultiPolygon]
 }
 
-public struct Geometry: Codable {
-    public var type: String
+public enum _Geometry {
+    private enum CodingKeys: String, CodingKey {
+        case type
+        case coordinates
+        case geometries
+    }
+    
+    case Point(coordinates: CLLocationCoordinate2D)
```

Comment:
`coordinate`, not `coordinates`

Reply:
Also, original `struct Point` refers to it's property as `coordinates`. So I vote for "coordinate**s**" :)